From fccf87968cad7a1781b348459e563d773fb842bd Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Sun, 27 Nov 2022 14:57:06 +0200
Subject: [PATCH] [SW-105118] greco: increase sram

Due to increase of fw_info, increase SRAM for Greco ARM

Change-Id: I5e9eeeeef6e48a44f7e586b05994685d034d138f
Signed-off-by: Guy Nisan <gnisan@habana.ai>
---

diff --git a/arch/greco/arm/common.h b/arch/greco/arm/common.h
index fb99d33..143dadc 100644
--- a/arch/greco/arm/common.h
+++ b/arch/greco/arm/common.h
@@ -11,7 +11,7 @@
 #include <sizes.h>
 
 #define PPBOOT_SIZE			0x6000		/* 24KB */
-#define PREBOOT_SIZE			0x14000		/* 80KB */
+#define PREBOOT_SIZE			0x15000		/* 84KB */
 
 #define HL_SP_SRAM_SIZE			SZ_64K
 
